google/cloud/pubsub/version.h

```diff
@@ -42,40 +42,12 @@ namespace pubsub {
  * symbol becomes `pubsub::v1::Foo`.
  */
 inline namespace GOOGLE_CLOUD_CPP_PUBSUB_NS {
-/**
- * The Cloud pubsub C++ Client major version.
- */
-int constexpr VersionMajor() {
-  return GOOGLE_CLOUD_CPP_PUBSUB_CLIENT_VERSION_MAJOR;
-}
-
-/**
- * The Cloud pubsub C++ Client minor version.
- */
-int constexpr VersionMinor() {
-  return GOOGLE_CLOUD_CPP_PUBSUB_CLIENT_VERSION_MINOR;
-}
-
-/**
- * The Cloud pubsub C++ Client patch version.
- */
-int constexpr VersionPatch() {
-  return GOOGLE_CLOUD_CPP_PUBSUB_CLIENT_VERSION_PATCH;
-}
-
-/// A single integer representing the Major/Minor/Patch version.
-int constexpr Version() {
-  static_assert(::google::cloud::version_major() == VersionMajor(),
-                "Mismatched major version");
-  static_assert(::google::cloud::version_minor() == VersionMinor(),
-                "Mismatched minor version");
-  static_assert(::google::cloud::version_patch() == VersionPatch(),
-                "Mismatched patch version");
-  return ::google::cloud::version();
-}
 
-/// The version as a string, in MAJOR.MINOR.PATCH+gitrev format.
-std::string VersionString();
+using ::google::cloud::version;
+using ::google::cloud::version_major;
+using ::google::cloud::version_minor;
+using ::google::cloud::version_patch;
+using ::google::cloud::version_string;
```

Comment:
Given the choice, I slightly lean toward not importing these into the pubsub namespace. They're not *needed* in this namespace. Having "version" functions in separate namespaces suggests that the version numbers might be different, when they won't be. Having them in two places makes people have to decide which to use. And we could safely add them to the pubsub  namespace later if we needed to, as opposed to never being able to remove them.

But then there is the argument for consistency w/ the other libraries. Either is fine with me. Your call.

Reply:
Removed. We can add them later, but when these libraries become GA there is no way to remove them.